common/protos/xyz/block/ftl/schema/v1/schema.proto

```diff
@@ -329,6 +330,12 @@ message MetadataCalls {
   repeated Ref calls = 2;
 }
 
+message MetadataCommit {
+  optional Position pos = 1;
```

Comment:
As discussed over DM, would be great to have the Git repo here, and maybe rename this to MetadataVCS